src/client/opamCommands.ml

```diff
@@ -668,8 +668,9 @@ let show =
                 OpamListCommand.field_names
               ^". Multiple fields can be separated with commas, in which case \
                 field titles will be printed; the raw value of any opam-file \
-                field can be queried by suffixing a colon character (:), e.g. \
-                $(b,--field=depopts:).")
+                field can be queried by combinig with $(b,--raw). For backward \
+                compatibilty, querying fieldi by suffixing a colon character \
```

Comment:
```suggestion
                compatibility, querying fields by suffixing a colon character \
```
(2 typos)
Not sure it's worth mentioning the `backwards compatibility` bit too.